app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    vaccinated = True
```

Comment:
There is no need for this flag, in case you face a `VaccineError`, you could add a corresponding `return` statement at the same stage, there is no need for further code execution in this case